src/test/helpers.js

```diff
@@ -27,7 +27,7 @@ function makeFake(algs, useFl) {
 
   const inst = xs.reduce((o, alg) => {
     const fn = useFl ? fl[alg] : alg
-    if(!isRepFunc(alg) && fn) {
+    if((!isRepFunc(alg) || alg === 'empty') && fn) {
```

Comment:
Why is this needed?

Reply:
This was needed because of the way that i was calling empty off of the instance. This code makes sure that the is an empty func on the instance and the constructor, however now that we've changed it this is very likely not needed. I just didn't think to remove it